phy/utils/_misc.py

```diff
@@ -98,3 +100,17 @@ def _show_shortcuts(shortcuts, name=''):
     for name in sorted(shortcuts):
         print('{0:<40}: {1:s}'.format(name, _show_shortcut(shortcuts[name])))
     print()
+
+def _git_version():
```

Comment:
I'm not sure really. What would you suggest? This sort of relies on having command-line git installed, and it will basically return a random string... the test would basically look like the function itself?...